cumulus/parachain-template/node/src/service.rs

```diff
@@ -43,20 +43,23 @@ pub fn new_partial<RuntimeApi, Executor, BIQ>(
 	config: &Configuration,
 	build_import_queue: BIQ,
 ) -> Result<
-	PartialComponents<
-		TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<Executor>>,
-		TFullBackend<Block>,
-		(),
-		sc_consensus::DefaultImportQueue<
-			Block,
-			TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<Executor>>,
-		>,
-		sc_transaction_pool::FullPool<
-			Block,
+	(
+		PartialComponents<
 			TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<Executor>>,
+			TFullBackend<Block>,
+			(),
+			sc_consensus::DefaultImportQueue<
+				Block,
+				TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<Executor>>,
+			>,
+			sc_transaction_pool::FullPool<
+				Block,
+				TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<Executor>>,
+			>,
+			(Option<Telemetry>, Option<TelemetryWorkerHandle>),
 		>,
-		(Option<Telemetry>, Option<TelemetryWorkerHandle>),
-	>,
+		NativeElseWasmExecutor<Executor>,
```

Comment:
Last generic parameter in `PartialComponents` was designed to return custom stuff, we should be able to add `NativeElseWasmExecutor` there instead of adding a tuple on top.